index.html

```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
       <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hesitate to contact me.</p>
 
-      <nav id="headline-nav">
-        <ul>
-          <li><a href="https://github.com/khanh-devos"><span></span></a></li>
-          <li><a href=""></a></li>
-          <li><a href=""></a></li>
-          <li><a href=""></a></li>
-          <li><a href=""></a></li>
+      
+      <ul class="social-media">
+        <li><a href="https://github.com/khanh-devos"></a></li>
+        <li><a href="https://www.linkedin.com/in/khanh-than-trong-3b4344246/"></a></li>
+        <li><a href=""></a></li>
+        <li><a href="https://twitter.com/home?lang=en"></a></li>
+        <li><a href="mailto:dreamproperty@gmail.com"></a></li>
+      </ul>
+
+    </section>
+
+    <section id="works">
+      <div id="works-intro">
+        <h1>My Recent Works</h1>
```

Comment:
- [ ] Kindly use only one h1 tag in your web application. It is best SEO practice to use a single h1 tag per web page. Kindly change this to another heading tag `h2 - h5`.